src/core/dbus-service.c

```diff
@@ -767,29 +765,30 @@ static int bus_service_set_transient_property(
                         /* Disallow empty string for ExtraFileDescriptors.
                          * Unlike OpenFile, StandardInput and friends, there isn't a good sane
                          * default for an arbitrary FD. */
-                        if (fd < 0 || isempty(fdname) || !fdname_is_valid(fdname))
```

Comment:
Any reason we removed the fd < 0 check?

Reply:
dbus's native `h` type would never return invalid fds to us. So this isn't necessary.